src/SFA.DAS.Charities.Import.Functions/Infrastructure/ITimeProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SFA.DAS.Charities.Import.Functions.Infrastructure
+{
+    public interface ITimeProvider
```

Comment:
normally seen this as IDateTimeProvider so it matches the actual DateTime type rather time ITimeProvider

Reply:
Fixed